build/docker/Dockerfile.backend

```diff
@@ -116,6 +116,6 @@ COPY --chown=node:node ./src ./src
 COPY --chown=node:node ./types ./types
 
 # This should be run as node
-USER node
+# USER node
```

Comment:
Should we remove this rather than comment it out? Or if we do leave it, should we document why?

Reply:
_done_